utils/.npmignore

```diff
@@ -0,0 +1,13 @@
+node_modules/
```

Comment:
Same here WRT resources e.g. the PNGs

Reply:
Do you know if the PNGs will still show up on the readme on npm?

Reply:
I guess I'm not certain. We could test without them in the package and just update it if they don't show up.